### DISCUSSION TITLE
Extensions are not loaded into Stability Matrix

### DISCUSSION THREAD

**Discussion by Cahulo2008** [LykosAI/StabilityMatrix#860] [General]
I have had to install Forge independently of Stability Matrix because, although it says that the extensions are loaded, they do not appear in the list within the interface. Does this happen to someone else or do you know how to solve it? The reactor is an essential extension for me and I can't use it with the stability matrix.

**Comment by Alexiazul** [comment]
I had the same issue with Adetailer a vital extension for anyone that renders faces. Same solution I had to install Forge Neo independent. Also when I install any extension an Apply and Quit Forge stalls out. I have to restart from Stability Matrix. This bug makes Stability Matrix useless to me.
